app/src/main/java/io/mercury/coroutinesandbox/view/downloader/DownloadListActivity.kt

```diff
@@ -104,28 +103,22 @@ class DownloadListActivity : ComponentActivity() {
                     }
                 }
 
+                val downloaderState = remember { downloaderFeature.state }.collectAsState()
```

Comment:
You can delegate this so you don't have to reference `value` everywhere below.

i.e.
`val downloaderState by remember { downloaderFeature.state }.collectAsState()`